extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerConfig.java

```diff
@@ -86,7 +100,7 @@ public GeoServerConfig(
 		if (update) {
 			ConfigOptions.writeProperties(
 					propFile,
-					gsConfig);
+					gsConfigProperties);
 
 			System.out.println("GeoServer Config Saved");
```

Comment:
I can switch to a log statement. The system.out was there previously and I didn't think to change it